### PR TITLE
Added SystemStatus into "docker info" response

### PR DESCRIPTION
**Pull request by damianopezzotti**
Class com.spotify.docker.client.messages.Info was enriched with systemStatus that contains useful information for Swarm Clusters (e.g. cluster node information)

### CODE REVIEW

**Comment by davidxia** [comment]
@damianopezzotti Thanks for the PR. Any way to get a test for this too? :)

**Comment by damianopezzotti** [comment]
Sorry but without Swarm value is null...

Il venerdì 4 marzo 2016, David Xia <notifications@github.com> ha scritto:

> @damianopezzotti <https://github.com/damianopezzotti> Thanks for the PR.
> Any way to get a test for this too? :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spotify/docker-client/pull/359#issuecomment-192488978>
> .
>


-- 
Damiano Pezzotti
...................................... . .
Software Architect | Alambitco
mobile +39 333 2116258
fax +39 02 700432209
email damy@alambitco.com
web www.alambitco.com
flickr www.flickr.com/photos/damianopezzotti
linkedin www.linkedin.com/in/damianopezzotti
pgp id 0x12DFD989

File: src/main/java/com/spotify/docker/client/messages/Info.java [hunk 1]
```diff
@@ -222,14 +228,14 @@ public int hashCode() {
   @Override
   public String toString() {
     return String.format("Info{ containers = %d, images = %d, storageDriver = %s, "
```
Comment (davidxia) [inline src/main/java/com/spotify/docker/client/messages/Info.java]: We should use guava's MoreObjects.toStringHelper() here:

```java
    return MoreObjects.toStringHelper(this)
        .add("containers", containers)
        ...
        .toString();
```

**Comment by davidxia** [comment]
@damianopezzotti Other than my comment on `toString()`, LGTM.

File: src/main/java/com/spotify/docker/client/messages/Info.java (hunk 1 above)
Comment (damianopezzotti) [inline src/main/java/com/spotify/docker/client/messages/Info.java]: Info now uses `MoreObjects.toStringHelper()`

**Pull request merged**